### PR TITLE
Kokoro: Removing the use of Google Maven Central mirror

### PR DESCRIPTION
**Pull request by suztomo**
We decided to stop using Google Maven Central mirror but there are some configuration that still references it.

### CODE REVIEW

File: kokoro/continuous.bat [hunk 1]
```diff
@@ -5,9 +5,6 @@ set PATH=%JAVA_HOME%\bin;%PATH%
 
 cd github/cloud-opensource-java
 
-mkdir %USERPROFILE%\.m2
-copy settings.xml %USERPROFILE%\.m2
```
Comment (elharo) [inline kokoro/continuous.bat:9]: Should we remove settings.xml from this repo? It's not used, I think.

Comment (suztomo) [reply kokoro/continuous.bat:9]: It's being removed in this PR (the 3rd file)

**Comment by suztomo** [comment]
@elharo All checks passed. PTAL.

File: kokoro/continuous.bat (hunk 1 above)
Comment (elharo) [reply kokoro/continuous.bat:9]: show missed it because github doesn't deletes :-)

**Review by elharo — APPROVED** [review]

**Pull request merged**